corgi/monitor/management/commands/runmonitor.py

```diff
@@ -11,6 +11,6 @@ class Command(BaseCommand):
     def handle(self, *args, **options):
         if settings.UMB_BREW_MONITOR_ENABLED:
             try:
-                BrewUMBListener.consume()
+                BrewBuildUMBListener.consume()
```

Comment:
You added two new listeners but they aren't actually being started to consume the messages from their configured topics, no? Or am I missing something?